src/components/ble/DfuService.h

```diff
@@ -9,13 +9,13 @@
 #undef max
 #undef min
 
+#include <drivers/SpiNorFlash.h>
```

Comment:
our own headers should be included with `""` instead of `<>`
```suggestion
#include "drivers/SpiNorFlash.h"
```